backend/MASZ/Controllers/api/v1/Views/VerifiedEvidenceViewController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Discord;
+using MASZ.Enums;
+using MASZ.Models;
+using MASZ.Models.Database;
+using MASZ.Models.Views;
+using MASZ.Repositories;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MASZ.Controllers.api.v1.Views
+{
+    [ApiController]
+    [Route("api/v1/guilds/{guildId}/evidence/{evidenceId}/view")]
+    [Authorize]
+    public class VerifiedEvidenceViewController : SimpleController
+    {
+        public VerifiedEvidenceViewController(IServiceProvider serviceProvider) : base(serviceProvider)
+        {
+        }
+
+        public async Task<IActionResult> GetEvidenceView([FromRoute] ulong guildId, [FromRoute] int evidenceId)
+        {
+            await RequirePermission(guildId, DiscordPermission.Moderator);
+
+            VerifiedEvidence evidence = await VerifiedEvidenceRepository.CreateDefault(_serviceProvider, await GetIdentity()).GetEvidence(guildId, evidenceId);
+
+            IUser reported = await _discordAPI.FetchUserInfo(evidence.UserId, CacheBehavior.OnlyCache);
+            IUser moderator = await _discordAPI.FetchUserInfo(evidence.ModId, CacheBehavior.OnlyCache);
+
+            VerifiedEvidenceExpandedView view = new(evidence, reported, moderator);
+
+            // TODO: mappings
```

Comment:
what about this todo @Simply-Cods ?

Reply:
This is actually fully implemented and I just forgot to remove the comment